frontend/src/core/helpers.js

```diff
@@ -154,10 +154,10 @@ define(function(require){
       }
       var html = '<ul class="tag-container">';
       for (var i = 0; i < list.length; ++i) {
-        var tag = (key && list[i][key]) ? list[i][key] : list[i];
+        var tag = Handlebars.Utils.escapeExpression((key && list[i][key]) ? list[i][key] : list[i]);
```

Comment:
Would you mind switching out the ternary to help a bit with readability?
```js
var item = list[i];
var tag = Handlebars.Utils.escapeExpression(key && item[key] || item);
```

Reply:
I'd prefer even moving all logic to the item declaration, and just have the tag use escapeExpression
```
var item = list[i][key] || list[i];
var tag = Handlebars.Utils.escapeExpression(item);
```